src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingServiceTest.java

```diff
@@ -152,6 +176,53 @@ void initialBeforeYouStart_TwoRespondentDetails_shouldReturnMarkUp() throws Exce
                 .isEqualTo(expected);
     }
 
+    @Test
+    void generateJurisdictionCodesHtml() {
+        CaseData caseData = new CaseData();
+        addJurCodeToExistingCollection(caseData, DAG);
+        String expected = String.format(JUR_CODE_HTML, String.format(CASE_NAME_AND_DESCRIPTION_HTML, DAG,
+            JurisdictionCode.valueOf(DAG).getDescription()));
+        assertThat(et1VettingService.generateJurisdictionCodesHtml(caseData.getJurCodesCollection()))
+            .isEqualTo(expected);
```

Comment:
This one is perfect.  Although very minor, the way a dev will read this, the new line before the .isEqualTo gives pause for breath.  A bit like adding a comma in a sentence.
Worth taking that on board as you develop more great code too ;)

Reply:
Yes, updated the others as well.